.github/workflows/build_wheels_linux.yml

```diff
@@ -97,16 +107,34 @@ jobs:
           repository: ${{ inputs.test-infra-repository }}
           ref: ${{ inputs.test-infra-ref }}
           path: test-infra
+      - uses: actions/checkout@v3
```

Comment:
IIUC per comment below, the setup script will occur in aarch64 docker and we wouldn't need to checkout builder anymore right?

Reply:
@osalpekar Correct, this won't be needed once we use aarch64 docker. Also its probably best to move aarch64 build scripts from builder to test-infra repo. Hence this will not be required anyways